collection/lens.rkt

```diff
@@ -39,6 +67,21 @@
   (make-lens (λ (target) (nth target index))
              (λ (target value) (set-nth target index value))))
 
+(define (mapper-lens lens)
```

Comment:
One, because `mapper-lens` is already the name in the `lens` library (http://pkg-build.racket-lang.org/doc/unstable-lens/index.html#%28def._%28%28lib._unstable%2Flens%2Fmapper..rkt%29._mapper-lens%29%29), although that's still in the unstable lens collection, and second, because otherwise it might seem like it should be `(map-lens lens list)` instead of `(lens-view (mapper-lens lens) list)`. At least that's my reasoning.